examples/bri-3/src/bri/communication/capabilities/createBpiMessage/createBpiMessageCommand.handler.ts

```diff
@@ -28,10 +30,13 @@ export class CreateBpiMessageCommandHandler
       command.type,
     );
 
-    const newBpiSubject = await this.storageAgent.createNewBpiMessage(
+    const newBpiMessage = await this.storageAgent.createNewBpiMessage(
       newBpiMessageCandidate,
     );
 
-    return newBpiSubject.id;
+    // TODO: This is purely for example purposes. Will be expanded on as part of upcoming issues related to messaging.
```

Comment:
Can we be a little more specific as to what the TODO is here.

Reply:
Fixed by removing TODO as i ll be implementing this in a day or two.